CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Change Log
 
+## Dangerzone 0.4.0
+- Bug fix: Failed execution no longer produces an empty "safe" documents ([issue #214](https://github.com/freedomofpress/dangerzone/issues/214))
+- Feature: support multiple input documents in the CLI-version
```

Comment:
It's best if we use the word "unreleased" for these sections, so that users don't get confused. This is already merged in the `main` branch, so it will be fixed once we rebase.

Reply:
Makes sense.